packages/dashboard/src/NavigationEvent.ts

```diff
@@ -0,0 +1,34 @@
+import { makeEventFunctions } from '@deephaven/golden-layout';
+
+const NavigationEvent = Object.freeze({
```

Comment:
I would've done these as separate consts, but I guess this makes sense to namespace them because we have the `index.ts` importing all the consts from a bunch of other files. Good work.